README.md

```diff
@@ -20,6 +20,7 @@ Displaying data from external services in a pinned gist.
 - [bird-box](https://github.com/matchai/bird-box) - Update a pinned gist to contain the latest tweets of a Twitter user.
 - [waka-box](https://github.com/matchai/waka-box) - Update a pinned gist to contain your weekly WakaTime stats.
 - [strava-box](https://github.com/JohnPhamous/strava-box) - Update a gist to contain your YTD exercise metrics from Strava.
+- [music-box](https://github.com/jacc/music-box) - Update a gist to contain your weekly listening report on last.fm.
```

Comment:
Total nitpick: I think we may want to capitalize "Last.fm", as that's how they refer to themselves on their site.
```suggestion
- [music-box](https://github.com/jacc/music-box) - Update a gist to contain your weekly listening report on Last.fm.
```

Reply:
added!